_data/schedule.yml

```diff
@@ -1,5 +1,99 @@
 2016:
   dates: March 10-11 2016
+  programme:
+    - day: Wednesday evening, 9th March
+      schedule:
+        - time: 5:30 pm - 9:30 pm
+          event: Pre-conference drinks
+          description: >
+            Come join us for pre-conference drinks at
+            <a href="https://www.facebook.com/dropkickmurphyscpt/">Dropkick Murphy's</a>, sponsored by
+            <a href="http://scaleconf.org/sponsors/flickswitch.html">Flickswitch</a>.
```

Comment:
Can you make this URL relative? Just make it `/sponsors/flickswitch.html`, that way local dev will mean that it points to the right place.

We should go back and fix the rest in this file, but that's a job for later.